app/src/main/java/org/mozilla/focus/utils/AppConstants.java

```diff
@@ -43,4 +44,8 @@ public static boolean isNightlyBuild() {
         return BuildConfig.FLAVOR_product == FLAVOR_product_NIGHTLY;
     }
 
+    public static boolean isProductionBuild() {
```

Comment:
we are using focusWebkitReleasefor production build.
and previewWebkitFirebase for nightly build